environments/plbase/plbase-install.sh

```diff
@@ -21,6 +21,7 @@ yum -y install \
     ImageMagick           `# for PrairieDraw label images` \
     texlive               `# for PrairieDraw label images` \
     git \
+    texlive-dvipng \
```

Comment:
Those comments are actually no longer valid, so let's get rid of them. And it's a bit tidier to keep the texlive entries together.
```suggestion
    ImageMagick \
    texlive \
    texlive-dvipng \
    git \
```